tripal_chado/tripal_chado.views.inc

```diff
@@ -0,0 +1,377 @@
+<?php
+
+/**
+ * Describe various Tripal Core systems to Views
+ *
+ * @ingroup tripal
+ */
+function tripal_chado_views_data() {
+  $data = [];
+  // Job Management System.
+  // tripal_chado_views_data_jobs($data);

```

Comment:
Hmmm.... that code should not have been there. It's a duplicate of the code in the `tripal` module for creating the Jobs View.  I suspect it may have been there as a reference for creating the Custom Tables view and was not later removed.  I removed it and will push.